Extensions/hideavatars.js

```diff
@@ -0,0 +1,239 @@
+//* TITLE Hide Avatars **//
+//* VERSION 0.1.0 **//
+//* DESCRIPTION Hides avatars on a per-url basis **//
+//* DEVELOPER dlmarquis **//
+//* FRAME false **//
+//* BETA true **//
+
+XKit.extensions.hideavatars = new Object({
+
+	running: false,
+	
+	blognames: [],
+	
+	run: function() {
+
+		this.running = true;
+
+		XKit.tools.init_css("hideavatars");
```

Comment:
I think this needs a corresponding `XKit.tools.remove_css` in the `destroy` function.